pymc3/distributions/multivariate.py

```diff
@@ -255,56 +262,53 @@ def random(self, point=None, size=None):
         -------
         array
         """
-        if size is None:
-            size = tuple()
+        size = to_tuple(size)
+
+        param_attribute = "chol_cov" if self._cov_type == "chol" else self._cov_type
```

Comment:
```suggestion
        param_attribute = "chol_tau" if self._cov_type == "tau" else "chol_cov"
```

Reply:
That's a good idea.

Reply:
I reverted this change in de1fa05 commit, because I was not getting correct cholesky decomposition for `cov` in `cov` parametrization. It was only giving identity matrix (maybe tag.test_value). I have no idea how to use theano functions to compile for getting correct cholesky decomposition.

So, I ask for `cov` samples (not for its `chol_cov`) and use `np.linalg.cholesky` to generate samples.